### PR TITLE
Allow updates to stylis from being used in ES bundle

### PR DESCRIPTION
**Pull request by philpl**
Before, even when stylis had a new patch version, Webpack 2 and Rollup would use the ES Bundle, which includes stylis. This PR changes the ES Bundle to import stylis instead.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "build:lib": "babel --out-dir lib src",
     "build:flow": "flow-copy-source -v -i '{**/test/*.js,**/*.test.js}' src lib",
     "prebuild:dist": "rimraf dist/*",
-    "build:dist": "rollup -c && rollup -c --environment PRODUCTION",
+    "build:dist": "rollup -c && rollup -c --environment ESBUNDLE&& rollup -c --environment PRODUCTION",
```
Comment (k15a) [inline package.json]: little nitpick: space is missing before &&
I don't know if some shells have a problem with it.

Comment (philpl) [inline package.json]: Ah oops

**Comment by silverlight513** [comment]
Hey, I just tested this work out on one of my projects and there's a slight hiccup. If you use npm, everything works fine, but if you use yarn, the yarn.lock file in this project is forcing the use of old packages. I.e. stylis at version 2.0.3 in the lock file which doesn't have some lovely fixes I was looking to use.

It's an easy switch to use npm for all users but forcing some users not to use yarn isn't great. Would it be okay if I provide a PR with an updated yarn.lock file?

**Comment by mxstbr** [comment]
Oh wow that's a good catch @silverlight513, please submit a PR with the latest 2.x version!

**Comment by silverlight513** [comment]
I'll get on it once I get home from work :)

**Pull request merged**